lib/pure/distros.nim

```diff
@@ -146,6 +146,7 @@ template unameRelease(cmd, cache): untyped =
 template uname(): untyped = unameRelease("uname -o", unameRes)
 template release(): untyped = unameRelease("lsb_release -d", releaseRes)
 template hostnamectl(): untyped = unameRelease("hostnamectl", hostnamectlRes)
+template etcOs(): untyped = unameRelease("grep '^NAME' /etc/os-release", etcOsRes)
```

Comment:
The "grep" part doesn't seem necessary.

Reply:
it's better than `cat /etc/os-release`:
![](https://i.imgur.com/1D5R8do.png)